src/components/MultipleSelect/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react';
+import './index.scss';
+
+interface MultipleSelectProps {
+   lists: string[];
+   listSeleted: string[];
+   handleSelect: (item: string[]) => void;
```

Comment:
```suggestion
   options: string[];
   value: string[];
   onChange: (item: string[]) => void;
```

E update lại style chút cho nó nhé, hiện tại a thấy hơi cùi chút :smile: , Với lại đang không biết là có đang chọn gì không, phải click vào mới biết